### PR TITLE
Password create/restore tasks for go-server

### PR DESCRIPTION
**Pull request by macdiesel**
@edx/pipeline-team PTAL.

### CODE REVIEW

File: playbooks/library/xml [hunk 1]
```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# xml - Manage bits and pieces of XML files
+#
+# Copyright 2014, Red Hat, Inc.
+# Tim Bielawa <tbielawa@redhat.com>
+# Magnus Hedemark <mhedemar@redhat.com>
+#
+# This software may be freely redistributed under the terms of the GNU
```
Comment (feanil) [inline playbooks/library/xml:10]: This license concerns me.  @nedbat will this be an issue?

Comment (nedbat) [inline playbooks/library/xml:10]: This wiki page https://openedx.atlassian.net/wiki/display/OPEN/Licensing indicates that GPL v2 is not compatible with AGPL v3.  I don't know the details of why not.

**Comment by doctoryes** [comment]
Where does this leave us? Should we write our own XML Ansible module? (Seems out of scope.) Are there other xml modules? 
Maybe we write our own `gocd_xml` module that has a single function of adding admin users to an XML config?

**Comment by macdiesel** [comment]
@doctoryes can't pull this in as it is. I am going to ask the author about changing the license scheme. In the interim I am going to pull out the XML pieces, I just haven't rounded back to this yet.

**Comment by macdiesel** [comment]
I have filed an issue upstream to ask the current maintainer to update their license: https://github.com/cmprescott/ansible-xml/issues/86

**Comment by macdiesel** [comment]
FYI the license for this library is being updated and we should be able to pull this in soon.

https://github.com/cmprescott/ansible-xml/issues/86

**Comment by feanil** [comment]
Closing since we can't merge this due to license issues.

**Pull request closed without merging**